### DISCUSSION TITLE
Welcome to coctohug Discussions!

### DISCUSSION THREAD

**Discussion by raingggg** [raingggg/coctohug#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guydavis** [comment]
Looks good!  I'll give it a try when I get a chance.  Great work on all the translations!

**Comment by crovebrave** [comment]
I like the translations too!

**Reply by raingggg** [reply]
Thanks all you guys! Happy farming!

**Comment by bistpon** [comment]
supports the most forks and updates quickly！
